chrome-ux-report/src/Code.js

```diff
@@ -324,6 +344,7 @@ function getOriginDataset(request) {
   var origin = {};
   origin.lastUpdate = getDatasetUpdate(request);
   origin.url = validateUrl(request.configParams);
+  origin.url = origin.url.toLowerCase();
```

Comment:
aren't some urls case-sensitive?

Reply:
url's domain part is not case-sensitive. CrUX stores information at domain level.